mllib-local/hs_err_pid78410.log

```diff
@@ -0,0 +1,526 @@
+#
```

Comment:
dont think you meant to add these log files?

Reply:
thanks for catching this. removed.